core/src/test/java/com/radixdlt/rev2/nofaucet/CallPreviewWithoutFaucetTest.java

```diff
@@ -0,0 +1,185 @@
+            0,
+            GenesisConsensusManagerConfig.testingDefaultEmpty(),
+            ImmutableList.of(
+                new GenesisDataChunk.Validators(
+                    ImmutableList.of(
+                        new GenesisValidator(
+                            validatorKey,
+                            true,
+                            true,
+                            Decimal.ZERO,
+                            ImmutableList.of(),
+                            Address.virtualAccountAddress(validatorKey)))),
+                new GenesisDataChunk.Stakes(
+                    ImmutableList.of(Address.virtualAccountAddress(validatorKey)),
+                    ImmutableList.of(
+                        tuple(
+                            validatorKey,
+                            ImmutableList.of(
+                                new GenesisStakeAllocation(
+                                    UInt32.fromNonNegativeInt(0), Decimal.ONE)))))),
+            Decimal.ofNonNegative(0L),
+            ImmutableList.of());
+
+    return DeterministicTest.builder()
+        .addPhysicalNodes(PhysicalNodeConfig.createBatch(1, true))
+        .messageSelector(firstSelector())
+        .messageMutator(MessageMutator.dropTimeouts())
+        .addModules(extraModules)
+        .functionalNodeModule(
+            new FunctionalRadixNodeModule(
+                FunctionalRadixNodeModule.NodeStorageConfig.tempFolder(folder),
+                true,
+                FunctionalRadixNodeModule.SafetyRecoveryConfig.BERKELEY_DB,
+                FunctionalRadixNodeModule.ConsensusConfig.of(1000),
+                FunctionalRadixNodeModule.LedgerConfig.stateComputerNoSync(
+                    StateComputerConfig.rev2().withGenesis(genesisData))));
+  }
+
+  @Test
+  public void call_preview_works_without_faucet() throws ApiException {
+    // Parameters corresponding to the following struct passed as input:
+    //    FungibleResourceManagerCreateWithInitialSupplyInput {
+    //      owner_role: Default::default(),
+    //      track_total_supply: Default::default(),
+    //      divisibility: Default::default(),
+    //      resource_roles: Default::default(),
+    //      metadata: Default::default()
+    //      address_reservation: Default::default(),
+    //      initial_supply: Default::default(),
+    //    }
+
+    final var addressing = Addressing.ofNetwork(Network.INTEGRATIONTESTNET);
+    final var accountLockerCall =
+        new TransactionCallPreviewRequest()
+            .network(Network.INTEGRATIONTESTNET.getLogicalName())
+            .target(
+                new BlueprintFunctionTargetIdentifier()
+                    .packageAddress(addressing.encode(ScryptoConstants.RESOURCE_PACKAGE_ADDRESS))
+                    .blueprintName("FungibleResourceManager")
+                    .functionName("create")
+                    .type(TargetIdentifierType.FUNCTION))
+            .addArgumentsItem("4d220000")
```

Comment:
(Side note, irrelevant to this PR!) Hah oh - I forgot how awful this interface is. I believe there's a ticket in the backlog to allow using manifest syntax for this in a new endpoint...